Today/Controllers/ReminderDetailViewController/ReminderDetailViewController.swift

```diff
@@ -54,9 +53,10 @@ class ReminderDetailViewController: UITableViewController {
             self.temporaryReminder = reminder
             self.editButtonItem.isEnabled = true
         }
-        navigationItem.title = isNew ? NSLocalizedString("Add Reminder", comment: "add reminder nav title") : NSLocalizedString("Edit Reminder", comment: "edit reminder nav title")
+        navigationItem.title = isNew ? NSLocalizedString(Strings.addReminderString, comment: Strings.addReminderNavTitleString) : NSLocalizedString(Strings.editReminderString, comment: Strings.editReminderNavTitleString)
```

Comment:
Have you created a localized file for strings ? 
And what steps we need to take if we have to add one other language support with minimal or no changes in existing code.

Reply:
Yes for English i have created a file named "Strings".
So if i have to add support for another language then i can create another file, lets say for German, named "GermanStrings". 
There i can define " static let addReminderGermanString = *Add Reminder in german* "

Reply:
It means you will update the code everywhere. To check the language before setting the string to any UIComponent ?

Reply:
Yeah that would be the case and quite a bad one. 
Another way i read once was localization. They stored such string files differently.